src/main/java/com/avispl/symphony/dal/logitech/collabos/common/LogitechCommand.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2024 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.logitech.collabos.common;
+
+/**
+ * LogitechCommand
+ *
+ * @author Harry / Symphony Dev Team<br>
+ * Created on 7/5/2024
+ * @since 1.0.0
+ */
+public enum LogitechCommand {
+	DEVICE_INFO("api/v1/device"),
+	INSIGHTS_ROOM("api/v1/insights/room"),
+	INSIGHTS_DEVICE("api/v1/insights/device"),
+	PERIPHERALS_INFO("api/v1/peripherals"),
+	;
+	private final String name;
```

Comment:
Maybe this could be renamed to uri instead of name, so it makes full sense outside, when we retrieve it from the LogitechCommand instances, as .getName() looks a bit confusing in terms of the http requests. Sure, it is the same functionally, but it kinda makes more sense :)